sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_counts_v1/query.sql

```diff
@@ -0,0 +1,300 @@
+WITH table_counts AS (
+  SELECT
+    'account_customers' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_account_customers_v1` FOR SYSTEM_TIME AS OF TIMESTAMP(
+      @as_of_date + 1,
+      'UTC'
+    )
+  UNION ALL
+  SELECT
+    'account_groups' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_account_groups_v1` FOR SYSTEM_TIME AS OF TIMESTAMP(
+      @as_of_date,
```

Comment:
All of the other time travel expressions in this query should also use `@as_of_date + 1` like the first one does, otherwise the data will be inconsistent.